src/main/java/spleetwaise/address/logic/LogicManager.java

```diff
@@ -96,41 +82,42 @@ public boolean isTransactionCommand(String commandText) {
 
     @Override
     public ReadOnlyAddressBook getAddressBook() {
-        return addressBookModel.getAddressBook();
+        return CommonModel.getInstance().getAddressBook();
```

Comment:
I wonder if it's better to have private class variable `commonModel = CommonModel.getInstance()` , and use `commonModel` throughout ?